framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1387,6 +1445,7 @@ public AnnotatedExecutableType shallowCopy(boolean copyAnnotations) {
 
             type.setElement(getElement());
             type.setParameterTypes(getParameterTypes());
+            type.computeVarargType();
```

Comment:
Same question here: would `computeVarargType` function properly when the returned values of `getParameterTypes()` are already adapted?